index.html

```diff
@@ -7,6 +7,13 @@
     <meta name="description" content="Resources and reviews for folx on parole">
     <link rel="icon" type="image/svg+xml" href="/favicon.png" />
     <link rel="apple-touch-icon" href="/apple-touch-icon-180x180.png">
+    <script type="text/javascript">
+      var environment = "%MODE%";
```

Comment:
Vite will replace this with the current vite mode (aka environment). https://vitejs.dev/guide/env-and-mode#html-env-replacement